packages/core/src/utils.js

```diff
@@ -990,6 +990,11 @@ export function toPathSchema(schema, name = "", rootSchema, formData = {}) {
     const _schema = retrieveSchema(schema, rootSchema, formData);
     return toPathSchema(_schema, name, rootSchema, formData);
   }
+
+  if (schema.hasOwnProperty("additionalProperties")) {
+    pathSchema.$additionalProperties = true;
```

Comment:
Can you rename `$additionalProperties` to `__rjsf_additionalProperties`?

I know we haven't been namespacing special properties like this before, but I think we should start doing it from now on whenever we add new ones.